src/main/java/frc/team2412/robot/util/auto/AutoLogic.java

```diff
@@ -469,6 +464,21 @@ public static Command setAngleSubwoofer() {
 				.withName("Auto - SetPivotSubwooferCommand");
 	}
 
+	public static Command visionLaunch2() {
```

Comment:
Maybe add a comment on the difference between visionLaunch and visionLaunch2 so that others know when to use which one and why these both exist.